x-pack/plugins/security_solution/cypress/tags.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const tag = {
+  SERVERLESS: '@serverless',
+  ESS: '@ess',
+  BROKEN_IN_SERVERLESS: '@brokenInServerless',
+};
```

Comment:
Could you please add a comment near each tag and explain when each should be used?